### ISSUE TITLE
Mod Organizer alters the function of my mouse

### ISSUE THREAD

**Issue by TheGreenDragon** [ModOrganizer2/modorganizer#527]
# The problem

After launching Mod Organizer the function of my mouse scroll wheel is altered. When pushing the mouse wheel left or right to scroll forward or back a page in Explorer, Chrome, all apps that allow the feature, the action is doubled causing a skip effect. Where I would go back one page on the internet, I now go back two. Instead of going up one level in a directory tree, I go up two. This persists until the computer is restarted. Relaunching the Logitech software or removing and reinstalling the mouse has no effect. 

# Environment

* Mod Organizer Version that exhibits the issue: All versions MO 1 and all versions MO 2
* Last Mod Organizer Verison that did not exhibit the issue (if applicable):N/A
* Desktop OS/version used to run Mod Organizer:Win 10 1803 (has occurred on all versions of W10)
* Logitech Gaming Software all versions (9.02.61 current)

# Details

I have had the issue for a very long time, just never bothered to report it. 

# Link to Mod Organizer logs

## USVFS
https://gist.github.com/TheGreenDragon/0ec8e904a249575a105eb63195a03ea2

## MO Interface
https://gist.github.com/TheGreenDragon/0ec8e904a249575a105eb63195a03ea2

**Comment by LostDragonist** [comment]
That's a fun one.  I have no clue how MO could even influence that.  So far, I've been unable to reproduce with the same version of Windows and Logitech Gaming Software with a G502 mouse on firmware 103.1.10.

**Comment by TheGreenDragon** [comment]
I agree, this will be difficult to pin down. If there is any information I can provide regarding the issue, please let me know.

**Comment by LostDragonist** [comment]
The model of your mouse (or previous mice) might be helpful.  I'd also make sure your mouse's firmware is up-to-date.

**Comment by TheGreenDragon** [comment]
I apologize, i wrote and rewrote this bug issue many times deciding if I was going to even post it at all. I swore I had the mouse model info in the original description. I will post it here and edit the original post for clarity.

**Comment by Al12rs** [comment]
Closing for organization and inability of reproducing, please reopen this if you can reproduce on 2.1.6+
